create_science_data/Create_example_level1_wfi_image.ipynb

```diff
@@ -0,0 +1,789 @@
+   "source": [
+    "## Make a somewhat random science data array\n",
+    "\n",
+    "Remember that Level 1 data are still in 3D ramp form"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2da7bba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data_frame = np.random.randint(low=0, high=65536,size=((4096,4096)), dtype=np.uint16)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea31d6de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data_frame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b278592d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.imshow(science_data_frame, origin=\"lower\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "31e5536b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make a small 3D cube\n",
+    "science_data_cube = np.array((science_data_frame, science_data_frame*2, science_data_frame*5))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b59317f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.shape(science_data_cube)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf176b2c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meta = {'telescope': 'ROMAN'}\n",
+    "meta['description'] = \"Example level 1 image mode ramp with random values\"\n",
+    "meta['author'] = 'Willy Wonka'\n",
+    "meta['origin'] = 'STSCI'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11531197",
+   "metadata": {},
+   "source": [
+    "### We can attempt to save this file using the minimal information that we have\n",
+    "\n",
+    "Validation errors should be returned since we dont have everything yet..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42e07da4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "level1['meta'] = meta\n",
+    "level1['data'] = science_data_cube"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f729516",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "af = asdf.AsdfFile()\n",
+    "af.tree = {'roman': level1}\n",
+    "af.write_to('level1_image_ramp.asdf')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0850c3f",
+   "metadata": {},
+   "source": [
+    "### Let's add in the rest of the information\n",
+    "\n",
+    "We'll cheat for now and pull it from the testing module for roman_datamodels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc4451fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from roman_datamodels.testing import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebe1fee9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "common_meta = utils.mk_common_meta()\n",
```

Comment:
When you show the roman_datamodels.testing.utils.mk_common_meta() function, it could be helpful to link to the code to show what it’s actually doing. It’s a little sneaky because while common_meta and all of its contents will behave like a dictionary you can’t just make a bunch of nested regular dictionaries. I think they’re going to want to see the roman_datamodels.stnode.Exposure() etc. calls to create the appropriately tagged objects.

Reply:
I dont really like linking back to this test code, but for now it's okay.